### PR TITLE
[zh-cn]: create the translation of `cookies.get()` method

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20967

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
与 #21051 类似，请先更新一下翻译（再好好检查一下）

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+title: cookies.get()
+slug: Mozilla/Add-ons/WebExtensions/API/cookies/get
+l10n:
+  sourceCommit: 43e3ff826b7b755b05986c99ada75635c01c187c
+---
+
+{{AddonSidebar}}
+
+{{WebExtAPIRef("cookies")}} API 的 **`get()`** 方法根据 Cookie 的名称和 URL 检索有关单个 Cookie 的信息。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:10]:
```suggestion
{{WebExtAPIRef("cookies")}} API 的 **`get()`** 方法根据 cookie 的名称和 URL 检索有关单个 cookie 的信息。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+如果给定 URL 存在多个具有相同名称的 Cookie，将返回路径最长的那个 Cookie。对于路径长度相同的 Cookie，将返回创建时间最早的那个。如果找不到匹配的 Cookie，则返回 `null`。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:12]:
```suggestion
如果给定 URL 存在多个具有相同名称的 cookie，将返回路径最长的那个 cookie。对于路径长度相同的 cookie，将返回创建时间最早的那个。如果找不到匹配的 cookie，则返回 `null`。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+let getting = browser.cookies.get(
+  details                // 对象
+)
+```
+
+## 参数
+
+- `details`
+
+  - : 一个 `object`，包含用于匹配要检索的 Cookie 的详细信息。它可以包含以下属性：
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:28]:
```suggestion
  - : 一个 `object`，包含用于匹配要检索的 cookie 的详细信息。它可以包含以下属性：
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+    - `firstPartyDomain` {{optional_inline}}
+
+      - : 一个 `string`，表示与要检索的 Cookie 关联的第一方域。如果浏览器启用了第一方隔离，则必须提供此属性。请参见[第一方隔离](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/cookies#第一方隔离)。。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:32]:
```suggestion
      - : 一个 `string`，表示与要检索的 cookie 关联的第一方域。如果浏览器启用了第一方隔离，则必须提供此属性。请参见[第一方隔离](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/cookies#第一方隔离)。。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+    - `name`
+
+      - : 一个 `string`，表示要检索的 Cookie 的名称。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:36]:
```suggestion
      - : 一个 `string`，表示要检索的 cookie 的名称。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+    - `partitionKey` {{optional_inline}}
+
+      - : 一个 `object`，表示包含 Cookie 的[存储分区](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/cookies#存储分区)。包括此对象和 `topLevelSite` 以从分区存储中获取 Cookie。否则，从未分区存储中返回 Cookie。此对象包含：
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:40]:
```suggestion
      - : 一个 `object`，表示包含 cookie 的[存储分区](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/cookies#存储分区)。包括此对象和 `topLevelSite` 以从分区存储中获取 cookie。否则，从未分区存储中返回 cookie。此对象包含：
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+        - `topLevelSite` {{optional_inline}}
+          - : 一个 `string`，表示包含 Cookie 的顶级站点存储分区的第一方 URL。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:43]:
```suggestion
          - : 一个 `string`，表示包含 cookie 的顶级站点存储分区的第一方 URL。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+    - `storeId` {{optional_inline}}
+
+      - : 一个 `string`，表示要查找的 {{WebExtAPIRef("cookies.CookieStore", "cookie 存储")}}的 ID（由 {{WebExtAPIRef("cookies.getAllCookieStores()")}} 返回）。默认情况下，将使用当前执行上下文的 Cookie 存储。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:47]:
```suggestion
      - : 一个 `string`，表示要查找的 {{WebExtAPIRef("cookies.CookieStore", "cookie 存储")}}的 ID（由 {{WebExtAPIRef("cookies.getAllCookieStores()")}} 返回）。默认情况下，将使用当前执行上下文的 cookie 存储。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+    - `url`
+
+      - : 一个 `string`，表示与要检索的 Cookie 关联的 URL。此参数可以是完整的 URL，在这种情况下，URL 路径之后的任何数据（例如查询字符串）将被忽略。如果此 URL 的[主机权限](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions)未在扩展的 [manifest 文件](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json)中指定，则 API 调用将失败。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:51]:
```suggestion
      - : 一个 `string`，表示与要检索的 cookie 关联的 URL。此参数可以是完整的 URL，在这种情况下，URL 路径之后的任何数据（例如查询字符串）将被忽略。如果此 URL 的[主机权限](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions)未在扩展的 [manifest 文件](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json)中指定，则 API 调用将失败。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+### 返回值
+
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，其会兑现一个包含有关 Cookie 的详细信息的 {{WebExtAPIRef('cookies.Cookie', 'Cookie')}} 对象，或者如果未找到 Cookie，则返回 `null`。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:55]:
```suggestion
一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，其会兑现一个包含有关 cookie 的详细信息的 {{WebExtAPIRef('cookies.Cookie', 'Cookie')}} 对象，或者如果未找到 cookie，则返回 `null`。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 示例
+
+此示例尝试获取与当前活动标签页的 URL 关联的名为“favorite-color”的 Cookie：
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:63]:
```suggestion
此示例尝试获取与当前活动标签页的 URL 关联的名为“favorite-color”的 cookie：
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+
+## 参数
+
+- `details`
+
+  - : 一个 `object`，包含用于匹配要检索的 cookie 的详细信息。它可以包含以下属性：
+
+    - `firstPartyDomain` {{optional_inline}}
+
+      - : 一个 `string`，表示与要检索的 cookie 关联的第一方域。如果浏览器启用了第一方隔离，则必须提供此属性。请参见[第一方隔离](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/cookies#第一方隔离)。。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:32]:
```suggestion
      - : 一个 `string`，表示与要检索的 cookie 关联的第一方域。如果浏览器启用了第一方隔离，则必须提供此属性。请参见[第一方隔离](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/cookies#第一方隔离)。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+    - `partitionKey` {{optional_inline}}
+
+      - : 一个 `object`，表示包含 cookie 的[存储分区](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/cookies#存储分区)。包括此对象和 `topLevelSite` 以从分区存储中获取 cookie。否则，从未分区存储中返回 cookie。此对象包含：
+
+        - `topLevelSite` {{optional_inline}}
+          - : 一个 `string`，表示包含 cookie 的顶级站点存储分区的第一方 URL。
+
+    - `storeId` {{optional_inline}}
+
+      - : 一个 `string`，表示要查找的 {{WebExtAPIRef("cookies.CookieStore", "cookie 存储")}}的 ID（由 {{WebExtAPIRef("cookies.getAllCookieStores()")}} 返回）。默认情况下，将使用当前执行上下文的 cookie 存储。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:47]:
```suggestion
      - : 一个 `string`，表示要查找的 {{WebExtAPIRef("cookies.CookieStore", "cookie 存储", "", 1)}}的 ID（由 {{WebExtAPIRef("cookies.getAllCookieStores()")}} 返回）。默认情况下，将使用当前执行上下文的 cookie 存储。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+
+    - `url`
+
+      - : 一个 `string`，表示与要检索的 cookie 关联的 URL。此参数可以是完整的 URL，在这种情况下，URL 路径之后的任何数据（例如查询字符串）将被忽略。如果此 URL 的[主机权限](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions)未在扩展的 [manifest 文件](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json)中指定，则 API 调用将失败。
+
+### 返回值
+
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，其会兑现一个包含有关 cookie 的详细信息的 {{WebExtAPIRef('cookies.Cookie', 'Cookie')}} 对象，或者如果未找到 cookie，则返回 `null`。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/get/index.md:55]:
```suggestion
一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，其会兑现一个包含有关 cookie 的详细信息的 {{WebExtAPIRef('cookies.Cookie', 'Cookie')}} 对象，或者如果未找到 cookie，则兑现 `null`。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**